src/glfs-cli.c

```diff
@@ -271,7 +271,7 @@ parse_options (struct cli_context *ctx)
         opterr = 0;
 
         while (true) {
-                opt = getopt_long (argc, argv, "o:", long_options,
```

Comment:
Fine. I will remove those and make the changes when you are done with your remaining reviews. Thanks.